readme.md

```diff
@@ -158,16 +159,16 @@ This would result in the code "RMWWL". To decrypt this code, you would simply ta
 
 The `substitution()` function in the `src/substitution.js` file has three parameters:
 
-- _input_ refers to the inputted text to be encoded or decoded.
-- _alphabet_ refers to substitution alphabet.
-- _encode_ refers to whether you should encode or decode the message. By default it is set to `true`.
+- _input_ is a string that refers to the inputted text to be encoded or decoded.
+- _alphabet_ is a string that refers to substitution alphabet.
+- _encode_ is a boolean that refers to whether you should encode or decode the message. By default, it is set to `true`.
 
 When building the function, keep the following constraints and rules in mind:
 
 - You are welcome to assume that _no additional symbols will be included as part of the input._ Only spaces and letters will be included.
-- Spaces should be maintained throughout.
-- Capital letters can be ignored.
-- The `alphabet` parameter must be string of exactly 26 characters. Otherwise, it should return `false`.
+- Spaces in the message should be maintained before and after encoding or decoding.
+- Encoding/decoding is case-insensitive (e.g., both "a" or "A" would be encoded to the same result).
```

Comment:
Would love to know if I interpreted "Capital letters can be ignored' correctly with this change.